src/module/utils/TwodsixRolls.ts

```diff
@@ -294,15 +294,20 @@ export class TwodsixRolls {
         * */
 
         //Handle special results
-        const diceValues:number[] = <number[]><unknown>roll.dice[0].results;
+
+        const diceResult = roll.dice[0].results.reduce((total:number, dice) => {
```

Comment:
This way we can get crit success/fail with advantage/disadvantage rolls as well.